extensions/wrapper/wrapper-common-mappers/src/main/java/de/sovity/edc/ext/wrapper/api/common/mappers/utils/UiAssetMapper.java

```diff
@@ -57,12 +52,20 @@ public UiAsset buildUiAsset(JsonObject assetJsonLd, String connectorEndpoint, St
         var creatorOrganizationName = JsonLdUtils.string(creator, Prop.Foaf.NAME);
         creatorOrganizationName = isBlank(creatorOrganizationName) ? participantId : creatorOrganizationName;
 
+        var description = JsonLdUtils.string(properties, Prop.Dcterms.DESCRIPTION);
+        String shortDescription = null;
+
+        if (description != null) {
+            shortDescription = markdownToTextConverter.extractText(description);
+        }
```

Comment:
I also recommed a private method "buildShortDescription(String description)". You can handle the nulls there